src/backend/src/main.ts

```diff
@@ -3,11 +3,12 @@ import 'dotenv/config';
 import { createNestApp } from './app.config';
 
 async function bootstrap() {
+
   // Initialize the Express based Nest application
   const { app } = await createNestApp();
   await app.init();
 
   // App listen to a port
-  await app.listen(process.env.PORT);
+  await app.listen(process.env.PORT || 3000);
```

Comment:
I would request you to add `.env` to your `backend` and get rid of `OR` statement here

```
ENV = development
PORT = 3000

API_NAME = Digital Privacy Impact Assessment API
API_VERSION = 0.0.1
```

Reply:
done